microsoft-edge/devtools/network/reference.md

```diff
@@ -1142,17 +1247,13 @@ To view an HTTP request's payload (query string parameters and form data), selec
 
    The request's payload is displayed, including a **Query String Parameters** section and a **Form Data** section:
 
-  ![The Query String Parameters section in the Payload tab](./reference-images/resources-headers-query-string-parameters.png)
+  ![The Query String Parameters section in the Payload tab](./reference-images/resources-headers-query-string-parameters.png)<!-- copy 2 of png is below -->
 
 In the **Console** tab of DevTools is the error:
 
 * `POST https://microsoftedge.github.io/Demos/devtools-network-reference/form-data-endpoint?hasfile=true 405 (Method Not Allowed) (anonymous) @ script.js:49`
 
-Or, if you use git bash, change directory to `/Demos/devtools-network-reference/`, run `npx http-server`, and then go to `http://localhost:8080/`, the **Console** error is:
-
-* `POST http://localhost:8080/form-data-endpoint?hasfile=true net::ERR_ABORTED 405 (Method Not Allowed) (anonymous) @ script.js:49`
-
-A harmless **Method Not Allowed** error appears in the **Console**, because there's no `form-data-endpoint` POST handler on the demo server.  The error is expected, because github.io hosting or the localhost npx server doesn't run application servers, only static files.
+This harmless **Method Not Allowed** error appears in the **Console**, because there's no `form-data-endpoint` POST handler on the demo server.  The error is expected, because github.io hosting or the localhost npx server doesn't run application servers, only static files.
```

Comment:
```suggestion
This harmless **Method Not Allowed** error appears in the **Console**, because there's no `form-data-endpoint` POST handler on the demo server.  The error is expected, because github.io hosting doesn't run application servers, only static files.
```

If we're not telling users to use npx in this section, we probably shouldn't tell them about it now.